imagetest/test_suites/ssh/image_ssh_test.go

```diff
@@ -90,3 +90,50 @@ func checkSudoGroup(client *ssh.Client, user string) error {
 	}
 	return nil
 }
+
+func TestHostKeysAreUnique(t *testing.T) {
+	vmname, err := utils.GetRealVMName("vm2")
+	if err != nil {
+		t.Fatalf("failed to get real vm name: %v", err)
+	}
+	pembytes, err := utils.DownloadPrivateKey(user)
+	if err != nil {
+		t.Fatalf("failed to download private key: %v", err)
+	}
+	client, err := createClient(user, fmt.Sprintf("%s:22", vmname), pembytes)
+	if err != nil {
+		t.Fatalf("user %s failed ssh to target host, %s, err %v", user, vmname, err)
+	}
+	bytes, err := getRemoteHostKey(client)
+	if err != nil {
+		t.Fatalf("failed to get host key from remote err %v", err)
+	}
+	remoteDiskEntries := utils.ParseHostKey(bytes)
```

Comment:
update to match suggestion for getRemoteHostKey:

```suggestion
	remoteDiskEntries, err := getRemoteHostKey(client)
	if err != nil {
		t.Fatalf("failed to get host key from remote err %v", err)
	}
```